interfaces/api/v1/blueprint/continuous_planning_routes.py

```
@@ -62,28 +79,93 @@ class ContinuePlanningRequest(BaseModel):
+        api_key = os.getenv("ANTHROPIC_API_KEY") or os.getenv("ANTHROPIC_AUTH_TOKEN")
+        if api_key:
+            try:
+                from infrastructure.ai.providers.anthropic_provider import AnthropicProvider
+                settings = Settings(
+                    api_key=api_key.strip(),
+                    base_url=os.getenv("ANTHROPIC_BASE_URL")
+                )
+                llm_service = AnthropicProvider(settings=settings)
+                print("✅ 使用 AnthropicProvider 初始化 LLM 服务")
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)